SECURITY.md

```diff
@@ -0,0 +1,122 @@
+
+Note that we should create a GitHub security advisory even if we choose to collaborate on a fix using an open issue or PR because we’ll want to publish the advisory when disclosing the vulnerability.
+
+### Request a CVE
+
+A CVE should be requested for all security vulnerabilities. Since we create a GitHub Security Advisory, we can [get a CVE from GitHub](https://docs.github.com/en/code-security/repository-security-advisories/about-github-security-advisories-for-repositories#cve-identification-numbers). As a backup, we can [get a CVE from MITRE](https://cveform.mitre.org/).
+
+### Develop a Patch
+
+Regardless of which approach we take, the VRT will prioritize developing a patch to fix the vulnerability. If the vulnerability is easily exploitable, remediation should take priority over all other work. Remediation should be completed within 7 days of the report being classified as a high-severity vulnerability.
+
+## Disclose
+
+We generally follow a 90-day disclosure timeframe, meaning that we ask that reporters give us 90 days to fix vulnerabilities before publicly disclosing them. 90 days should be viewed as an upper bound, and we should aim to remediate vulnerabilities as quickly as possible. In all cases, the disclosure timeline should be explicitly coordinated with the reporter, and we should arrange to publicly disclose the vulnerability simultaneously with the reporter. Our disclosure will include credit for the reporter if they so wish.
+
+In rare cases, it may be appropriate to delay public disclosure even after the patch has been published and deployed. However, since our source code is public, we should assume that attackers will quickly reverse-engineer the vulnerability from our patch and so err on the side of disclosing early.
+
+Our public disclosure should take the form of a published GitHub Security Advisory. Here is a template:
+
+```
+# Security advisory for $CVEID
+
+## Summary
+
+<!--
+A brief (as short as possible, about a paragraph) summary of the vulnerability using technical details. The goal of this is to allow the reader to do a quick assessment of what the bug is about.
+-->
+
+* **CVE**: $CVE-NUMBER
+* **Affected versions**: $VERSIONS
+* **Reporter**: $NAME $AFFILIATION
+
+## Severity
+<!--
+[Low, Medium, HIGH, **CRITICAL**] - Accompany your assessment with a motivation, and even a good attack scenario to explain the risk associated with the vulnerability.
+-->
+
+## Proof of Concept
+<!--
+Code or command lines. We want to offer a concrete, usable, and repeatable way for the reader to reproduce the issue you are raising so they can test fixes and mitigations.
+-->
+
+## Remediation and Mitigation
+<!--
+Known remediations. If one is a software update, note the version(s) that fix it.
+-->
+
+## Further Analysis
+<!--
+If you wish to add more context or information, we recommend adding it after the critical sections mentioned here. If you know the vulnerability has been exploited in the wild, add that here.
```

Comment:
I disagree with the last part of this, at least with its current wording. We should always indicate whether we know if the vulnerability has been exploited, and then take additional measures to reach out to affected users in the case where their data may have been compromised. Users probably won't be satisfied with just "the vulnerability has probably been exploited".

Reply:
What wording would you prefer? In practice, I think we'll rarely be able to rule out a vulnerability being exploited--the attacker could always have been really good at covering their tracks. Instead, we'll only ever be able to say either we know it has been exploited or we have no indication that it was exploited.

Reply:
Discussed with @BenHenning -- the wording in [the current version](https://github.com/oppia/oppia/pull/16470/files#diff-ac3050733d58345444c27d37c6247ef260ad992bcb02782feea43fb580106079R111-R120) is fine. Thanks!